proxies/sandbox/apiproxy/policies/AssignMessage.AddCors.xml

```diff
@@ -4,7 +4,7 @@
     <Set>
         <Headers>
             <Header name="Access-Control-Allow-Origin">{request.header.origin}</Header>
-            <Header name="Access-Control-Allow-Headers">origin, x-requested-with, accept, content-type, authorization, x-correlation-id, x-request-id</Header>
+            <Header name="Access-Control-Allow-Headers">origin, x-requested-with, accept, content-type, authorization, x-correlation-id, x-request-id, NHSD-Session-URID</Header>
```

Comment:
should this be lowercase like the other header values in here?

Reply:
Think you are right. Was just looking at the screenshot on try this API.

![image](https://user-images.githubusercontent.com/23194784/161801871-1d59c86f-48a7-4d78-b0fb-4ad10d0c7bed.png)

Seems like all mixed case headers have just been made lower. Fixing.